apps/dashboard/lib/auth/cookies.ts

```
@@ -112,9 +112,39 @@ export async function SetSessionCookie(params: {
+  const options = {
+    httpOnly: true,
+    secure: true,
+    sameSite: "strict",
+    path: "/",
+    maxAge: Math.floor((expiresAt.getTime() - Date.now()) / 1000),
+  };
+
+  let cookieString = `Path=${options.path}`;
+  
+  if (options.httpOnly) {
+    cookieString += `; HttpOnly`;
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)